tests/dummy/app/templates/identity/community-header/standalone.hbs

```diff
@@ -1,8 +1,19 @@
-{{wds-community-header model=model}}
+{{#wds-community-header model=model track=track}}
+	{{partial 'partials/community-header-wiki-content'}}
+{{/wds-community-header}}
 <div class="separator"></div>
-{{wds-community-header model=model class='color-5F1023'}}
+{{#wds-community-header model=model class='color-5F1023' track=track}}
+	{{partial 'partials/community-header-wiki-content'}}
+{{/wds-community-header}}
+
 <div class="separator separator-5F1023"></div>
-{{wds-community-header model=model class='color-787878'}}
+{{#wds-community-header model=model class='color-787878' track=track}}
+	{{partial 'partials/community-header-wiki-content'}}
+{{/wds-community-header}}
+
 <div class="separator separator-787878"></div>
-{{wds-community-header model=model class='color-FB7383'}}
+{{#wds-community-header model=model class='color-FB7383' track=track}}
+	{{partial 'partials/community-header-wiki-content'}}
```

Comment:
I'd rewrite it to a component
partials are an old construct and will most likely be deprecated in future